downstream/assemblies/platform/assembly-metrics-utility.adoc

```diff
@@ -0,0 +1,46 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-07-12
+
+ifdef::context[:parent-context-of-metrics-utility: {context}]
+
+:_mod-docs-content-type: ASSEMBLY
+
+[id="metrics-utility_{context}"]
+
+:context: metrics-utility
+
+= Subscription consumption
+
+The {PlatformNameShort} metrics utility tool (`metrics-utility`) is a command-line utility that is installed on a system containing an instance of {ControllerName}. 
+
+When installed and configured, `metrics-utility` gathers billing-related metrics from your system and creates a consumption-based billing report. Metrics-utility is especially suited for users who have multiple managed hosts and want to use consumption-based billing. Once a report is generated, it is deposited in a target location that you specify in the configuration file. 
+
+Metrics-utility collects two types of data from your system: configuration data and reporting data. 
+
+The configuration data includes the following information:
+
+* Version information for {ControllerName} and for the operating system 
+* Subscription information
+* The base URL
+
+The reporting data includes the following information:
+
+* Job name and ID
+* Host name
+* Inventory name 
+* Organization name 
+* Project name 
+* Success or failure information
+* The date and time the report was generated.  
```

Comment:
Style guidelines say to Write lists so that all or none of the items start with complete sentences. But, when a list contains a combination of complete sentences and fragments to include periods after all fragments and sentences in that list. 

Suggest either turning the last bullet into a fragment and removing the period or adding a period for all list items to conform to style.

Reply:
updated